[2016-04-25 16:17:42] - [DEBUG] - [pylxca.pylxca_api.lxca_connection] - Establishing Connection
[2016-04-25 16:17:49] - [WARNING] - [py.warnings] - /usr/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/connectionpool.py:768: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

[2016-04-25 16:17:50] - [WARNING] - [py.warnings] - /usr/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/connectionpool.py:768: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

[2016-04-25 16:17:53] - [DEBUG] - [pylxca.pylxca_api.lxca_api] - Connection to LXCA Success
[2016-04-25 16:18:08] - [WARNING] - [py.warnings] - /usr/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/connectionpool.py:768: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

[2016-04-25 16:18:40] - [WARNING] - [py.warnings] - /usr/local/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg/requests/packages/urllib3/connectionpool.py:768: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

[2016-04-25 16:18:55] - [ERROR] - [pylxca.pylxca_api.lxca_api] - Exception HTTPSConnectionPool(host='10.241.106.216', port=443): Max retries exceeded with url: /chassis/ (Caused by ReadTimeoutError("HTTPSConnectionPool(host='10.241.106.216', port=443): Read timed out. (read timeout=3)",)) Occurred while calling REST API for object cmms
